app/views/content/online_chat_privacy_notice.md

```diff
@@ -42,39 +44,37 @@ We’ll also analyse and consolidate data from support requests to help us under
 
 ### How we protect your data
 
-The Zendesk customer service management system is hosted in accordance with its [regional data hosting policy](https://support.zendesk.com/hc/en-us/articles/360022185194?_ga=2.9135733.1989263323.1628099773-531922291.1627980374).
-
+The Zendesk customer service management system is hosted in accordance with its [regional data hosting policy](https://support.zendesk.com/hc/en-us/articles/360022185194).
 Zendesk is required to process and protect your personal data under strict conditions. These are stated in a written agreement between us and Zendesk, as required by the UK General Data Protection Regulation (GDPR) and UK Data Protection Act 2018 (DPA 2018).
 
 ### How long we keep your data
 
-We keep your data only for as long as needed for the uses listed above.
-
-We’ll need to support some people considering applying for teacher training for many years. We’ll delete the data we hold on you 3 years after you first contact us.
+We’ll delete the data we hold on you 3 years after you first contact us.
 
 ### Your rights
+Under the DPA 2018 and UK GDPR and the lawful basis of public task in the public interest, you have the right to:
 
-Under the DPA 2018 and UK GDPR and the lawful basis of public interest, you have the right to:
+_ find out what data we hold about you
+_ be informed about how your data is being used
+_ access personal data
+_ have incorrect or incomplete data updated
+_ object to how your data is processed
+_ restrict the processing of your data in certain circumstances
+_ object to direct marketing
+_ object to automated decision making and profiling
```

Comment:
```suggestion
- find out what data we hold about you
- be informed about how your data is being used
- access personal data
- have incorrect or incomplete data updated
- object to how your data is processed
- restrict the processing of your data in certain circumstances
- object to direct marketing
- object to automated decision making and profiling
```

Reply:
Thanks Despo, I was having trouble deciding between `-` and `*` and ended up with `_`!